GDEdit Web/GDE.Web/Shared/NavMenu.razor

```diff
@@ -1,43 +1,69 @@
-<div class="top-row pl-4 navbar navbar-dark">
-    <a class="navbar-brand" href="">GDE.Web</a>
-    <button class="navbar-toggler" @onclick="@ToggleNavMenu">
-        <span class="navbar-toggler-icon"></span>
-    </button>
-</div>
+@using GDE.Web.Data
+<link rel="stylesheet" href="css/navbar.css"/>
+<style>
+	@switch (GlobalVariables.CurrentSection)
```

Comment:
Preferably make a separate function where it returns the value to set the hue to and call it to avoid
```csharp
statement;
break;
```
on each case.